src/extras/curves/CatmullRomCurve3.js

```diff
@@ -84,6 +84,7 @@ THREE.CatmullRomCurve3 = ( function() {
 		function ( p /* array of Vector3 */ ) {
 
 			this.points = p || [];
+			this.closed = false;
```

Comment:
Right now it feels to me that `.closed` is a property of `CatmullRomCurve3` rather than `Curve`. If `.closed` is a property of `Curve` then all curves should support using this property which we currently don't...